lib/webdriveragent.js

```diff
@@ -104,6 +105,15 @@ class WebDriverAgent {
       await fixXCUICoordinateFile(this.bootstrapPath);
     }
 
+    if (this.prebuildWDA) {
+      // first do a build phase
+      log.debug('Pre-building WDA before launching test');
+      this.usePrebuiltWDA = true;
+      this.xcodebuild = await this.createXcodeBuildSubProcess(true);
+      await this.startXcodebuild(true);
+      delete this.xcodebuild;
```

Comment:
we typically set fields to `null` rather than deleting them. but i'll read on and see if there's a good reason for doing it this way.